sdk/core/azure-core/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 1.0.0-beta.4 (Unreleased)
 
+### Breaking Changes
+- Removed `DateTime::operator Duration()`.
```

Comment:
nit: add a new line between heading and list (linter)

Reply:
Good suggestion, I did not see any linter messages, but let's take care of it during the next readme update, or before the release.

Reply:
BTW, beta3's New Features section also needs a newline.